knife-solo.gemspec

```diff
@@ -31,6 +30,6 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'coveralls'
 
   s.add_dependency 'chef',     '>= 10.20'
-  s.add_dependency 'net-ssh',  '~> 2.7', '< 3.0'
-  s.add_dependency 'erubis',   '~> 2.7.0'
+  s.add_dependency 'net-ssh',  '~> 2.6'
```

Comment:
Chef 12.7.2 declares `"net-ssh", ">= 2.9", "< 4.0"` in https://github.com/chef/chef/blob/12.7.2/chef.gemspec#L27, could knife-solo, too? Or is there a reason why 2.6 is now declared?